openmetadata-ui/src/main/resources/ui/cypress/e2e/Features/DataConsumerRole.spec.js

```diff
@@ -24,13 +24,13 @@ import {
   createEntityTable,
   createSingleLevelEntity,
   hardDeleteService,
-} from '../../common/entityUtils';
+} from '../../common/EntityUtils';
```

Comment:
do we have this file? I can see the file name as `entityUtils`

Reply:
Yes, I've taken care of renaming the file